### ISSUE TITLE
Blog improvement for users 'readability' and 'reachability in mobile'

### ISSUE THREAD

**Issue by SreyleakH** [neos/Neos.NeosIo#127]
1. News details content area is **too wide from left to right**. While reading the post it make me feel that I would never finish the content.  What do you think of making news details display in 2 columns, by having the news content on the left and list of previous news on the right instead of at down below?

2. The content in general, is a bit too small. Are you all fine reading the text?

3. In mobile: I think EVENT and BLOG are the informations you want users to find most. If I am right, can we make these on the top menu of the menu?

<img width="1277" alt="screen shot 2016-11-24 at 15 50 16" src="https://cloud.githubusercontent.com/assets/23726789/20607625/479ec210-b27a-11e6-9276-5b061f4af2a5.png">

<img width="887" alt="screen shot 2016-11-24 at 15 50 57" src="https://cloud.githubusercontent.com/assets/23726789/20607626/4f2226a8-b27a-11e6-8953-6ee045b0fa63.png">

**Comment by aertmann** [comment]
This is a bit ironic, when I first implemented the blog I did it in a way where the text was larger and the lines were about 80 characters long and the latest/related news were on the right. However @Inkdpixels changed this for aesthetic reasons. Personally I agree that the text is too small and the lines are too long and thus not accessible according to accessibility guidelines. @kdambekalns has previously said the text was small as well.

Btw. also noticed that related news and latest posts are shown differently.
![image](https://cloud.githubusercontent.com/assets/903567/20675460/53cf38ca-b55a-11e6-8fb6-44026062d7f4.png)
